### ISSUE TITLE
Whitelist some sites for the container

### ISSUE THREAD

**Issue by totokaka** [containers-everywhere/contain-google#54]
When logging into my google account with OneLogin, the login process hangs if I have this extension enabled.

My workaround is this:
- Open gmail in the google container - it will be logged out
- Disable the extension
- Login to gmail in the container with OneLogin - OneLogin is now open inside the container
- Gmail logs in and is still in the container
- Enable the extension

I believe this process could be streamlined by allowing the onelogin domain to stay in the container when opened from the container. I believe the risk involved in this is very low.

I do not think OneLogin should be hard coded in the application, but that we rather should accept a whitelist of domains that are allowed to stay inside the container from settings.

I can create a PR for this in a while.

**Comment by herpiko** [comment]
I think there is no need for whitelist. We can make it works by defining some oauth related paths that need to be available to perform such login. Make it an option just like `Ignore Youtube`.

**Comment by totokaka** [comment]
I do not believe your PR will fix this issue, as this relates to logging into Google with SSO.
As for the whitelist: I believe it makes more sense to let a few third party sites into the google container, than to let a few google URI out of the container.

**Comment by isaaclw** [comment]
>  I believe it makes more sense to let a few third party sites into the google container

That would be my preference also.

**Comment by situplastik** [comment]
Totally agree with @tlaundal . Here an issue asking for the same #113

**Comment by situplastik** [comment]
PR #114 resolve that problem @Perflyst
